src/wallet/coinselection.h

```diff
@@ -174,7 +174,7 @@ struct OutputGroup
  * change_cost = effective_feerate * change_output_size + long_term_feerate * change_spend_size
  *
  * @param[in] inputs The selected inputs
- * @param[in] change_cost The cost of creating change and spending it in the future. Only used if there is change. Must be 0 if there is no change.
+ * @param[in] change_cost The cost of creating change and spending it in the future. Must be positive if there is change. Must be 0 if there is no change.
```

Comment:
maybe keep both the "Only used if" and "must be positive" info
```diff
- * @param[in] change_cost The cost of creating change and spending it in the future. Only used if there is change. Must be 0 if there is no change.
+ * @param[in] change_cost The cost of creating change and spending it in the future.
+ *                        Only used if there is change, in which case it must be positive.
+ *                        Must be 0 if there is no change.
```

Reply:
done